rust/foxglove/src/channel.rs

```diff
@@ -56,6 +56,16 @@ impl Schema {
             data: data.into(),
         }
     }
+
+    /// Returns a JSON schema for the specified type.
+    pub fn json_schema<T: schemars::JsonSchema>() -> Self {
+        let json_schema = schemars::schema_for!(T);
+        Self::new(
+            std::any::type_name::<T>(),
+            "jsonschema",
+            serde_json::to_vec(&json_schema).expect("Failed to serialize schema"),
```

Comment:
I know this predates the PR, but is `expect` what we want here? Is it OK because of some guarantees schemars makes?

Reply:
Yeah, this is a good use case for a panic. This error would only ever happen if the schemars crate was completely bugged. Developer error, not a runtime error.